warehouse/util.js

```diff
@@ -18,9 +18,31 @@ const isObject = value => {
   );
 };
 
+const isJson = value => {
+  return value != null && (typeof value === "object" || Array.isArray(value));
```

Comment:
Why are arrays considered to be JSON?

Reply:
according to warehouses, arrays can only be stored in json column type. This mapping is about under which column type incoming event columns fit in database